aws/data_source_aws_iam_policy_document.go

```diff
@@ -30,6 +30,11 @@ func dataSourceAwsIamPolicyDocument() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"override_json_list": {
```

Comment:
There was some discussion around `override_json_list` or `override_jsons`. I think the `_list` is cleaner, but completely open for discussion.